test/TestCases.test.js

```diff
@@ -29,9 +29,9 @@ describe("TestCases", function() {
 			webpack(options, function(err, stats) {
 				if(err) return done(err);
 				if(stats.hasErrors()) return done(new Error(stats.toString()));
-				var testFile = path.join(outputDirectory, "test.js");
+				var testFile = path.join(testDirectory, "test.js");
```

Comment:
This fixes an apparent bug. As far as I can tell, this whole section was setup to allow us to test output js, but wasn't used up till now. Here, it's simply looking in the wrong directory for the original test file.